File: 151003/Baranouski/RV_Project/src/main/java/by/bsuir/RV_Project/dto/requests/converters/AuthorRequestConverter.java
package by.bsuir.RV_Project.dto.requests.converters;

import by.bsuir.RV_Project.domain.Author;
import by.bsuir.RV_Project.dto.requests.AuthorRequestDto;
import org.mapstruct.Mapper;

@Mapper(componentModel = "spring")
public interface AuthorRequestConverter {
    Author fromDto(AuthorRequestDto author);
}

File: 151003/Yagnnish/Lab1/Dtos/RequestDtos/RequestConverters/AuthorRequestConverter.cs
using lab_1.Domain;

namespace lab_1.Dtos.RequestDtos.RequestConverters
{
    public class AuthorRequestConverter:BaseRequest<Author,AuthorRequestDto>
    {
        public  Author FromDto (AuthorRequestDto dto,long? id)=>new Author(id,dto?.login,dto?.password,dto?.firstname,dto?.lastname);
    }
}
